btcjson/walletsvrresults.go

```diff
@@ -159,3 +159,17 @@ type GetBestBlockResult struct {
 	Hash   string `json:"hash"`
 	Height int32  `json:"height"`
 }
+
+// BalancesResult models the details data from the `getbalances` command.
+type BalancesResult struct {
```

Comment:
Maybe call this `BalanceDetails` or something else, in order to differentiate it from `GetBalancesResult`. What do you think?

Reply:
Sounds good! I have renamed it to `BalanceDetailsResult` as many other structs seem to keep the *Result prefix.